src/engine/openrpc/methods/payload.yaml

```diff
@@ -94,6 +94,7 @@
       required:
         - executionPayload
         - blockValue
+      additionalProperties: false
```

Comment:
Sorry can you revert the changes to the engine api methods? We don't have I/O tests for these yet and so I don't want to add further constraints to them without verifying they are correct.

Reply:
sure, updated